src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double printArea();

```

Comment:
It does not print area, instead, it calculates in and returns the `double` value. So let's rename it accordingly.